dnf-behave-tests/features/microdnf/install3.feature

```diff
@@ -0,0 +1,25 @@
+@no_installroot
+Feature: microdnf install command on packages
+
+
+Background:
+  Given I delete file "/etc/dnf/dnf.conf"
+    And I delete file "/etc/yum.repos.d/*.repo" with globs
+
+
+
```

Comment:
Three newlines here and in some other feature files. Two is enough...

Reply:
Still the three newlines... :slightly_smiling_face: Here and in `install[4-6].feature`.